src/main/webapp/app/overview/header-course.component.ts

```diff
@@ -49,4 +52,13 @@ export class HeaderCourseComponent implements OnChanges {
             }
         }
     }
+
+    showGoToCourseManagementButton() {
+        const courseManagementPage = this.router.url.startsWith('/course-management');
+        return !courseManagementPage && this.course.isAtLeastTutor;
+    }
+
+    redirectToCourseManagement() {
+        this.router.navigate([`/course-management/${this.course.id}`]);
```

Comment:
```suggestion
        this.router.navigate(["course-management", this.course.id]);
```

I would use this notation for `navigate`.

Reply:
Done